components/hidden_data_box/wikis/valorant/hidden_data_box_custom.lua

```diff
@@ -0,0 +1,66 @@
+---
+-- @Liquipedia
+-- wiki=valorant
+-- page=Module:HiddenDataBox/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local String = require('Module:StringUtils')
+local Tier = mw.loadData('Module:Tier')
+local Variables = require('Module:Variables')
+
+local BasicHiddenDataBox = require('Module:HiddenDataBox')
+local CustomHiddenDataBox = {}
+
+function CustomHiddenDataBox.run(args)
+	BasicHiddenDataBox.addCustomVariables = CustomHiddenDataBox.addCustomVariables
+	BasicHiddenDataBox.setWikiVariableForParticipantKey = CustomHiddenDataBox.setWikiVariableForParticipantKey
+	args.liquipediatier = Tier.number[args.liquipediatier]
+	return BasicHiddenDataBox.run(args)
+end
+
+function CustomHiddenDataBox.addCustomVariables(args, queryResult)
+	Variables.varDefine('tournament_parent_name', Variables.varDefault('tournament_parentname'))
+	Variables.varDefine('tournament_tier', Variables.varDefault('tournament_liquipediatier'))
+	Variables.varDefine('tournament_tiertype', Variables.varDefault('tournament_liquipediatiertype'))
+	Variables.varDefine('tournament_date', Variables.varDefault('tournament_enddate'))
+	Variables.varDefine('tournament_sdate', Variables.varDefault('tournament_startdate'))
+	Variables.varDefine('tournament_edate', Variables.varDefault('tournament_enddate'))
+	Variables.varDefine('edate', Variables.varDefault('tournament_enddate'))
```

Comment:
only edate, not sdate?

Reply:
https://liquipedia.net/rocketleague/index.php?title=Template:HiddenDataBox&action=edit 🤷

Reply:
🤷